test/e2e.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+
+TOP_DIR=$(readlink -f `dirname "${BASH_SOURCE[0]}"` | grep -o '.*/oshinko-webui')
+PROJECT=$(oc project -q)
+
+WEBUI_START_XVFB=${WEBUI_START_XVFB:-true}
+WEBUI_TEST_IMAGE=${WEBUI_TEST_IMAGE:-}
+WEBUI_TEST_SECURE=${WEBUI_TEST_SECURE:-false}
+WEBUI_TEST_LOCAL_IMAGE=${WEBUI_TEST_LOCAL_IMAGE:-true}
+
+# If you're doing a test of the secure webui, default user/pass is "developer/developerpass"
+# If you need to change it you can set these
+WEBUI_TEST_SECURE_USER=${WEBUI_TEST_SECURE_USER:-}
+WEBUI_TEST_SECURE_PASSWORD=${WEBUI_TEST_SECURE_PASSWORD:-}
+
+# This is all for dealing with registries. External registry requires creds other than the current login
+WEBUI_TEST_INTEGRATED_REGISTRY=${WEBUI_TEST_INTEGRATED_REGISTRY:-}
+WEBUI_TEST_EXTERNAL_REGISTRY=${WEBUI_TEST_EXTERNAL_REGISTRY:-}
+WEBUI_TEST_EXTERNAL_USER=${WEBUI_TEST_EXTERNAL_USER:-}
+WEBUI_TEST_EXTERNAL_PASSWORD=${WEBUI_TEST_EXTERNAL_PASSWORD:-}
+
+WEBUI_TEST_RESOURCES=${WEBUI_TEST_RESOURCES:-$TOP_DIR/tools/resources.yaml}
+if [ -z "$WEBUI_TEST_IMAGE" ]; then
+    if [ "$WEBUI_TEST_LOCAL_IMAGE" == true ]; then
+        WEBUI_TEST_IMAGE=oshinko-webui:latest
+    else
+        WEBUI_TEST_IMAGE=docker.io/radanalyticsio/oshinko-webui
+    fi
+fi
+
+function print_test_env {
+    echo Using image $WEBUI_TEST_IMAGE
```

Comment:
hmm, this thing should print whether it's a secure test or not and whether or not the user/pass has been overridden